command.go

```diff
@@ -42,7 +42,7 @@ func NewCommand(
 		},
 	}
 
-	var cmdListTests = &cobra.Command{
+	var cmdList = &cobra.Command{
 		Use:           "list",
 		Aliases:       []string{"ls"},
 		Short:         "List registered tests.",
```

Comment:
Should this short description also be updated since list will list both tests and suites now?

Reply:
Yeah this should be updated.